litellm/proxy/proxy_server.py

```diff
@@ -352,6 +352,24 @@ def _get_pydantic_json_dict(pydantic_obj: BaseModel) -> dict:
         return pydantic_obj.dict()
 
 
+def get_custom_headers(*,
+                       model_id: Optional[str] = None,
+                       cache_key: Optional[str] = None,
+                       api_base: Optional[str] = None,
+                       version: Optional[str] = None,
+                       model_region: Optional[str] = None
+                       ) -> dict:
+    exclude_values = {'', None}
+    headers = {
+        'x-litellm-model-id': model_id,
+        'x-litellm-cache-key': cache_key,
+        'x-litellm-model-api-base': api_base,
+        'x-litellm-version': version,
+        '"x-litellm-model-region': model_region
+    }
+    return {key: value for key, value in headers.items() if value not in exclude_values}
```

Comment:
you should check if the value is not None here too

Reply:
`None` will be checked with `value not in exclude_values` which contain None and empty string.